code/modules/biomatter_manipulation/solidifier.dm

```diff
@@ -2,8 +2,9 @@
 //This machine converts liquid biomatter to solid one(sheets)
 //Working with this also required bio protection cloths
 
-#define BIOMATTER_PER_SHEET 		10
+#define BIOMATTER_PER_SHEET 		1
 #define CONTAINER_PIXEL_OFFSET 		6
+#define BIOMATTER_SHEETS_PER_TIME  5 // X sheets per second
```

Comment:
// X sheets per 2 seconds

Reply:
Whoops. I've fixed the typo in next commit.